webpack/__mocks__/foremanReact/common/forceSingleton.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Force a single instance to protect from code duplication.
+ *
+ * WARNING: Code duplications happen because of an issue with the build process,
+ *          so this method might be removed once the issue would be fixed.
+ *          See: https://projects.theforeman.org/issues/27195
+ *
+ * @param  {String}   key    A unique-key to save the instance.
+ * @param  {Function} create A function to create an instance.
+ * @return {*}               Single Instance,
+ *                           returned by the create method or from the cache.
+ */
+const forceSingleton = (key, create) => {
```

Comment:
Where do we import `forceSingleton`? I do not see it  being used anywhere...

Reply:
Ops, it's a mistake, I will delete it,